src/components/ContentHighlights/CourseHighlightRoutes.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Container } from '@edx/paragon';
+import Hero from '../Hero';
+import { ROUTE_NAMES } from '../EnterpriseApp/constants';
+import ContentHighlightSet from './ContentHighlightSet';
+import ContentHighlightsDashboard from './ContentHighlightsDashboard';
+
+const CourseHighlightRoutes = ({ enterpriseSlug }) => {
+  const baseContentHighlightPath = `/${enterpriseSlug}/admin/${ROUTE_NAMES.contentHighlights}`;
+  return (
+    <>
+      <Helmet>
+        <title>Highlights</title>
+      </Helmet>
```

Comment:
We'll probably want to define `<title>` differently when on the `ContentHighlightsDashboard` route versus the `ContentHighlightSet` route.

Example: "Highlights" vs. "Example Highlight Title - Highlights"